organization/annual-reports/financial-report-2019-2020.md

```diff
@@ -187,4 +187,200 @@ Gifts may only be accepted without prior approval when:
 
 Hospitality included in participation in an approved mission, conference, workshop or other meeting does not require separate approval.
 
-#### 1.5 Communication with stakeholders, community and general public
+### 1.5 Communication with stakeholders, community and general public
+
+The Foundation for Public Code is an open organization with collaboration and transparency at its core. Our goal is to ensure that everything we do is open and publicly accessible. We document all of our activities, processes and organizational information so that our stakeholders, the community, and the general public can contribute to our work. The documentation of our work is core to our ability to deliver on our mission together with the community.
+
+We continuously publish updates about our work via our website, social media, and mailing lists, in order to keep our audience informed. We also attend events throughout the year to engage with our stakeholders and raise awareness about public code.
+
+### 1.6 Risk paragraph
+
+The Foundation for Public Code was wholly philanthropically funded during the reporting period. We have received donations from three different parties:
+
+* Dia Holding B.V. (Arnout Schuijff)
+* DC Butterfield and AD Rubio Memorial Foundation (Stewart Butterfield)
+* Elkes Foundation (Josh Elkes)
+
+These philanthropists share our commitment to our mission. To prevent undue influence, all donations are un-earmarked and go into our general operating budget, in compliance with our status as a Dutch public benefit organization (ANBI).
+
+During this start-up phase, we try to look two years ahead with our financial planning.  The organization  has low financial risk: there are no debts and no assets. There is a loan agreement (anti-kraak) for the office, which incurs no rent. In the second quarter of 2020, we transitioned to remote-first working in response to government recommendations during the Covid-19 crisis. When the loan agreement expires at the end of June 2021, the contract can be extended on a monthly basis until a tenant is found. All our activities are location independent, meaning the organization can continue to function in a remote-first context. Nevertheless, some budget has been set aside starting from the third quarter of 2021 in case a new office space is needed.
```

Comment:
Nit-pick: extra space in "The organization  has low financial risk" .... not a blocker at all, as it won't render any differently in a browser.

Reply:
(there are a few other double-spaces in the middle of sentences on the page as well ... again, not a big deal)

Reply:
Done, thanks!